c_glib/arrow-glib/column.cpp

```diff
@@ -322,7 +322,8 @@ garrow_column_get_field(GArrowColumn *column)
   } else {
     const auto arrow_column = garrow_column_get_raw(column);
     auto arrow_field = arrow_column->field();
-    return garrow_field_new_raw(&arrow_field);
+    return garrow_field_new_raw(&arrow_field,
+                                garrow_column_get_data_type(column));
```

Comment:
We need to call `g_object_unref()` for `GArrowDataType *` returned by `garrow_column_get_data_type()`. Because `garrow_column_get_data_type()` creates a new `GArrowDataType`.

Reply:
I've fixed it.